secondIife.js

```diff
@@ -0,0 +1,37 @@
+var CarLot = (function (NewCarLot) {
+		// grab all inputs globally 
+		var carHolder = document.getElementsByClassName("carCard");
+		var newInputField = document.getElementById("inputField");
+		var descriptionInput = document.getElementById("Description");
+		var targetToEdit = "";
+		// itterates through the car cards 
+	NewCarLot.activateEvents = function() {
+	    for (var i = 0; i < carHolder.length; i++) {
+	      var containerEl = carHolder[i];
+	      // add event listener to car cards
+	      containerEl.addEventListener("click", function(event) {
+	            targetToEdit = this.childNodes[4];
+	            newInputField.focus();
+	            newInputField.value = targetToEdit.innerHTML;
+	            this.classList.toggle("border");
+	           	$(this).css("background-color", "#C5CADF");
```

Comment:
@ostrander-nathan was just curious why you added a class to the div, and then manually changed the background color.  It works, so you don't **need** to _fix_ it, just something to think about.